crates/api/src/tx.rs

```diff
@@ -0,0 +1,14 @@
+//! Kitsune2 transport module.
+
+use crate::{AgentId, BoxFut, K2Result, OpId};
+
+/// Trait for implementing a transport module for exchanging messages
+/// between agents.
+pub trait Transport: 'static + Send + Sync + std::fmt::Debug {
+    /// Send a request for ops to a peer.
+    fn send_op_request(
+        &mut self,
+        op_id: OpId,
+        dest: AgentId,
+    ) -> BoxFut<'static, K2Result<()>>;
```

Comment:
This is what the call currently looks like in my wip pr.

```suggestion
    /// Notify a remote peer module within a space. This is a fire-and-forget
    /// type message. The future this call returns will indicate any errors
    /// that occur up to the point where the message is handed off to
    /// the transport backend. After that, the future will return `Ok(())`
    /// but the remote peer may or may not actually receive the message.
    pub async fn send_module(
        &self,
        peer: Url,
        space: SpaceId,
        module: String,
        data: bytes::Bytes,
    ) -> BoxFut<'static, K2Result<()>>;
```